src/site/content/en/learn/pwa/caching/index.md

```diff
@@ -1,11 +1,168 @@
 ---
 title: Caching
 description: >
-  A good PWA has all the necessary assets cached in the client. You will learn how to leverage the Cache Storage API for storing these assets
+  With the Cache Storage API, you can ask the browser to download, store, delete or update assets on the device. The engine can then serve these assets on the device without needing a network request.
```

Comment:
I'm hesitant of removing the "you can ask" because I still want to convey that the developers have agency with the API.

Maybe rewriting it as:
 "You can use the Cache Storage API to download, store, delete or update assets on the device. Then these assets can be served on the device without needing a network request."

or since this is the intro now that I think about it I don't know why it only speaks about the cache storage api, maybe:
"Caches are powerful tools that enable fast loading and offline experiences on the web. In this chapter you'll learn best practices and how to use the he Cache Storage API to download, store, delete or update assets on the device. Then serve those assets  without needing a network request."

Reply:
I prefer your first suggestion. 

So many writers describe APIs as 'powerful' that the word has lost its meaning smacks of hyperbole.